lib/buildFiles.js

```diff
@@ -29,13 +29,37 @@ function buildFiles(dictionary, platform) {
     throw new Error('Build path must end in a trailing slash or you will get weird file names.')
   }
 
+  var warningText = `
+    templates will be removed in Style Dictionary 3.0, please change your config to use formats.
```

Comment:
check in which release this is going to be merged, otherwise the message needs to be updated

Reply:
changing to the "in the future" wording you provided.